apps/src/sites/studio/pages/admin_standards/import_standards.js

```diff
@@ -0,0 +1,25 @@
+$(document).ready(function() {
+  $('#import-standards').click(function(e) {
+    var courseCode = $('#select option:selected').val();
+    var url =
+      'http://curriculumbuilder.herokuapp.com/metadata/' +
```

Comment:
do we want to be pulling from heroku here or codecurricula.com? It feels like we might move away from heroku at some point so it might be better to point at codecurricula.com?